pkgs/development/libraries/clutter/default.nix

```diff
@@ -19,12 +19,20 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gtk3 ];
   nativeBuildInputs = [ pkg-config gobject-introspection ];
-  propagatedBuildInputs =
-    [ libX11 libGL libGLU libXext libXfixes libXdamage libXcomposite libXi cogl pango
-      atk json-glib gobject-introspection libxcb libinput libgudev libxkbcommon
-    ];
+  propagatedBuildInputs = [
+    cogl pango atk json-glib gobject-introspection
```

Comment:
While changing this, could you please make the dependencies one per line so that we do not have to introduce another diff mess in the future?